src/main/java/og_spipes/service/SPipesExecutionService.java

```diff
@@ -61,8 +61,7 @@ public String serviceExecution (
             Map<String, String> params
     ) throws SPipesEngineException {
         String serviceUrl = engineUrl + "/service";
-        params.put("id", functionId);
-        params.put("_pConfigURL", pConfigURL);
+        params.put("_pId", functionId);
```

Comment:
I believe that you cannot remove `_pConfigURL` variable until  you solve the issue in https://github.com/kbss-cvut/s-pipes-editor-ui/issues/27 .